network.env.dist

```diff
@@ -1,18 +1,14 @@
 #Unique Coinbase addresses
 
-PRIME_COINBASE=0x00114a47a5d39ea2022dd4d864cb62cfd16879fc
-REGION_0_COINBASE=0x0d79b69c082e6f6a2e78a10a9a49baedb7db37a5
-REGION_1_COINBASE=0x3bccBe6C6001C46874263169dF887Cbf5C3580d6
-REGION_2_COINBASE=0x5a457339697cb56e5a9bfa5267ea80d2c6375d98
-ZONE_0_0_COINBASE=0x1930e0b28d3766e895df661de871a9b8ab70a4da
-ZONE_0_1_COINBASE=0x246ae82bb49e9dda583cb5fd304fd31cc1b69790
-ZONE_0_2_COINBASE=0x2e82bec9c7e47564b9e89b5ab989c0002373c497
-ZONE_1_0_COINBASE=0x421bc7323295c6b7f2f75fc4c854d4fb600e69e7
-ZONE_1_1_COINBASE=0x4d6605da9271f8bcea42326a07f3c43f7f67a431
-ZONE_1_2_COINBASE=0x59630c586ede320c8d7759f38373564f5f9faf20
-ZONE_2_0_COINBASE=0x6b70b802661a87823a9cb16a8a39763bc8e11de4
-ZONE_2_1_COINBASE=0x70820eb5e384b65caf931e85224601f3203ecd20
-ZONE_2_2_COINBASE=0x7f8a6306426b57d13f1e3975d377574e18d7a7a3
+ZONE_0_0_COINBASE=0x04a3e45aa16163F2663015b6695894D918866d19
```

Comment:
Please drop this commit

Reply:
Yes this was the result of a sloppy rebase, I've dropped the commit and tested it.